docs/tools/cli/project.rst

```diff
@@ -276,3 +276,30 @@ Removes a user with the supplied email address from the project.
 .. code:: shell
 
   avn project user-remove --project my-project alice@example.com
+
+
+Request project SBOM
+--------------------
+
+SBOM reports are generated per project and can be downloaded as long as the necessary permissions are set for the project. You can get the SBOM report download link for a project using the following command:
+
+``avn_project_generate_sbom``
+
+``avn project generate-sbom``
```

Comment:
```suggestion
```

Reply:
Just deleting the duplicate.

Reply:
in that case I think I will keep the other one with the spaces in between words :slightly_smiling_face: